packages/app-store/routing-forms/lib/incompleteBooking/actionDataSchemas.ts

```diff
@@ -0,0 +1,10 @@
+import type { z } from "zod";
+
+import { routingFormIncompleteBookingDataSchema as salesforceRoutingFormIncompleteBookingDataSchema } from "@calcom/app-store/salesforce/zod";
+import { IncompleteBookingActionType } from "@calcom/prisma/enums";
+
+const incompleteBookingActionDataSchemas: Record<IncompleteBookingActionType, z.ZodType<any>> = {
```

Comment:
The incomplete booking action contains a `data` field. We're using Zod to validate